.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots / Videos**
+If applicable, add screenshots to help explain your problem.
+
+**Platform:**
```

Comment:
Also capture networking setup? Not using the words "networking setup" since people are confused what that means

Reply:
Wdym? like how ethernet cables are connected between a radio and switch or more like IP configurations.

Reply:
Both, I think? Only if required. Main goal here is to make sure people are using a radio and network switch and not doing something stupid, so whatever makes sure of that